docs/ebb.html

```diff
@@ -2677,6 +2677,17 @@ <h2><a name="states"></a>Initial I/O pin configuration</h2>
         and defaults to an off state at reset; see the <code><a href="#SR">SR</a></code> command.
         </p>
         
+        <p>Pins B1, B0, B2 and B3 are not 5-volt tolerant and any voltage above about 3.6V will damage them.
+        Pins B4, B5, B6 and B7 are 5-volt tolerant and will not be damaged by voltages up to 6.0V.
```

Comment:
IIRC, the datasheet says 5.5 V.

Reply:
I did check the datasheet before putting these numbers in here. It has 6.0V listed as the abs max for any I/O pin with respect to Vss. Would you like me to de-rate this value down to 5.5V to provide a less risky upper end?

Reply:
My datasheet does list 6.0 V in the Absolute Maximum table, but it describes them elsewhere as "5.5 V tolerant."

I guess that this is technically correct.